public/src/components/Manager/Manager.vue

```diff
@@ -35,25 +35,56 @@
     </v-bottom-nav>
   </v-app>
 </template>
-
 <script>
 export default {
-  name: 'Manager',
-  created () {
+  name: "Manager",
```

Comment:
aren't we using singlequotes in js?

Reply:
yeah the prettier i'm using messed this up. i'm introducing a prettierrc file in a different pull request

Reply:
i merged in #62 and i'm going to re-run it against this file to get the single quotes init

Reply:
fixed